components/login/passwordReset/src/index.scss

```
@@ -0,0 +1,5 @@
+@import '~@s-ui/theme/lib/index';
+
+.sui-LoginPasswordReset {
```

```suggestion
.mt-LoginPasswordReset {
```

🍆  should be like this, right?

---

This component is intended to be global. reusable for all verticals. Maybe we can think if the prefix "sui-" is the best... but coupling it to "mt" is not the idea.

---

you're absolutely right! I missed we're on this repo! 😅